src/Item.js

```diff
@@ -10,8 +10,8 @@ var Item = function(parameters) {
     if(parameters.options) {
     	var _this = this;
     	parameters.options.forEach(function(value) {
-      	_this.Options[value] = { '1/1': '1' };
-    	})
+			_this.Options[value] = { '1/1': '1' };
```

Comment:
just notes, I know its not your code.
we probably shouldn't default the options for all items, I will be updating this with the menu options.

also the use of _this is not really ideal.